metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java

```diff
@@ -79,6 +79,15 @@
      * Iceberg table type.
      */
     public static final String ICEBERG_TABLE_TYPE = "ICEBERG";
+    /**
+     * Iceberg table type.
+     */
+    public static final String COMMON_VIEW = "common_view";
```

Comment:
Is this lower-cased?

Reply:
yes, this is a key in table paramters, e.g.
```
parameters: {
previous_metadata_location: "",
metadata_location: "s3n://netflixxee.metadata.json",
common_view: "true",
table_type: "VIRTUAL_VIEW"
},
```
```